src/ontology/reports/iri-sources.md

```diff
@@ -0,0 +1,12 @@
+These are the versions of the ingested resources that are currently being ingested into Mondo.  
+Note that the specific version listed here might be only partially integrated into Mondo (the integration is ongoing).
+
+##  Versions of ingested resources
+
+- http://purl.obolibrary.org/obo/mondo/sources/2025-04-26/doid.owl
+- http://purl.obolibrary.org/obo/mondo/sources/2025-02-26/icd10cm.owl
+- http://purl.obolibrary.org/obo/mondo/sources/2025-02-26/icd10who.owl
+- http://purl.obolibrary.org/obo/mondo/sources/2025-02-26/icd11foundation.owl
+- http://purl.obolibrary.org/obo/mondo/sources/2025-02-26/ncit.owl
+- http://purl.obolibrary.org/obo/mondo/sources/2025-02-26/omim.owl
+- http://purl.obolibrary.org/obo/mondo/sources/2025-04-26/ordo.owl
```

Comment:
There is an important misunderstanding here.. these are the version Iris of the components, which is fine I guess, but what we really need is the version iri of the source!

Reply:
what does source mean to you, e.g. mirror? is the version IRI of a component changed in the process of being converted from mirror to component? i've seen this happen in some step

Reply:
ah, ok so the component is the "version" that gets changed in processing. 

@joeflack4 do you have time to do this today?

Reply:
@matentzn is half correct. ODK changes the IRI when we download the mirror. See: [slack](https://monarchinitiative.slack.com/archives/C03AW548J8M/p1747419520741479?thread_ts=1747418752.898679&cid=C03AW548J8M)

We could make secondary goals and download the mirrors a second time, but that adds a lot more build time / complexity than I hoped.

@matentzn @twhetzel Let me know when you guys come up with a solution that is to your liking~

Reply:
We have to put this on hold until ODK is released, I have provided a fix there: https://github.com/INCATools/ontology-development-kit/pull/1239

We will soon release.

Reply:
Nice!

Reply:
If all goes as planned 31st May

Reply:
I saw that @matentzn opened this PR concurrent w/ the release:
- #841 

But it looks like he's still working on getting that latest release to work in that PR. So I think that is a prerequisite to this PR.

Reply:
@matentzn do you have updates on when the `mondo-repo` will be updated to ODK 1.6? In Slack, this Draft PR was mentioned https://github.com/monarch-initiative/mondo-ingest/pull/841. The ODK update is a dependency for this PR and this PR is a dependency to address this user request https://github.com/monarch-initiative/mondo/issues/8949. Can you work with @sabrinatoro to help manage feedback on this user submitted issue? Thanks!

Reply:
Sure, that sounds fine. Good to hear the other issues were sorted. I'll convert https://github.com/monarch-initiative/mondo-ingest/pull/841 to ready for review.

Do you anticipate that the IRIs will be available for the next Mondo release and/or can you respond to the user with your timeline https://github.com/monarch-initiative/mondo/issues/8949?